--- rhythmbox-0.12.8.old/plugins/iradio/iradio-initial.pls	2010-03-25 00:10:12.000000000 +0000
+++ rhythmbox-0.12.8/plugins/iradio/iradio-initial.pls	2010-06-25 16:57:49.000000000 +0100
@@ -9,13 +9,13 @@ file2=http://network.absoluteradio.co.uk
 title2=Absolute Radio (Broadband)
 genre2=Pop
 
-file3=http://network.absoluteradio.co.uk/core/audio/ogg/live.pls?service=vx
-title3=Absolute Xtreme (Modem)
-genre3=Modern Rock
-
-file4=http://network.absoluteradio.co.uk/core/audio/ogg/live.pls?service=vxbb
-title4=Absolute Xtreme (Broadband)
-genre4=Modern Rock
+file3=http://network.absoluteradio.co.uk/core/audio/ogg/live.pls?service=al
+title3=Absolute Radio dabbl (Modem)
+genre3=General
+
+file4=http://network.absoluteradio.co.uk/core/audio/ogg/live.pls?service=albb
+title4=Absolute Radio dabbl (Broadband)
+genre4=General
 
 file5=http://network.absoluteradio.co.uk/core/audio/ogg/live.pls?service=vc
 title5=Absolute Classic Rock (Modem)
@@ -47,34 +47,34 @@ title11=WKNC 88.1 FM (NC State) (High Qu
 genre11=Music
 
 # http://www.nrk.no/lyd/
-file14=http://media.hiof.no/streams/m3u/nrk-p1-172.ogg.m3u
-title14=NRK P1 (Norway)
+file12=http://media.hiof.no/streams/m3u/nrk-p1-172.ogg.m3u
+title12=NRK P1 (Norway)
+genre12=General
+
+file13=http://media.hiof.no/streams/m3u/nrk-p2-172.ogg.m3u
+title13=NRK P2 (Norway)
+genre13=General
+
+file14=http://media.hiof.no/streams/m3u/nrk-petre-172.ogg.m3u
+title14=NRK P3 (Norway)
 genre14=General
 
-file15=http://media.hiof.no/streams/m3u/nrk-p2-172.ogg.m3u
-title15=NRK P2 (Norway)
-genre15=General
-
-file16=http://media.hiof.no/streams/m3u/nrk-petre-172.ogg.m3u
-title16=NRK P3 (Norway)
-genre16=General
+file15=http://media.hiof.no/streams/m3u/nrk-alltid-nyheter-172.ogg.m3u
+title15=NRK Alltid Nyheter (Norway)
+genre15=
+
+file16=http://media.hiof.no/streams/m3u/nrk-mpetre-172.ogg.m3u
+title16=NRK mP3 (Norway)
+genre16=
 
-file17=http://media.hiof.no/streams/m3u/nrk-alltid-nyheter-172.ogg.m3u
-title17=NRK Alltid Nyheter (Norway)
+file17=http://media.hiof.no/streams/m3u/nrk-alltid-klassisk-172.ogg.m3u
+title17=NRK Alltid Klassisk (Norway)
 genre17=
 
-file18=http://media.hiof.no/streams/m3u/nrk-mpetre-172.ogg.m3u
-title18=NRK mP3 (Norway)
+file18=http://media.hiof.no/streams/m3u/nrk-stortinget-172.ogg.m3u
+title18=NRK Stortinget (Norway)
 genre18=
 
-file19=http://media.hiof.no/streams/m3u/nrk-alltid-klassisk-172.ogg.m3u
-title19=NRK Alltid Klassisk (Norway)
-genre19=
-
-file20=http://media.hiof.no/streams/m3u/nrk-stortinget-172.ogg.m3u
-title20=NRK Stortinget (Norway)
-genre20=
-
-file21=http://www.wbur.org/listen/feed/ogg.m3u
-title21=wbur.org - Boston's NPR news source
-genre21=News
+file21=http://www.kwur.com/icecast/kwurogg.m3u
+title21=KWUR 90.3 FM St. Louis Underground Radio
+genre21=
